apps/projects/app/utils/apollo-client.js

```diff
@@ -1,16 +1,35 @@
 import ApolloClient from 'apollo-boost'
+import { useAragonApi } from '../api-react'
+import { GITHUB_TOKEN_REVOKED } from '../store/eventTypes'
+import { STATUS } from '../utils/github'
 
-export const initApolloClient = token =>
-  new ApolloClient({
+export const useApolloClient = () => {
+  const {
+    api,
+    appState: {
+      github: { token } = { token: null }
+    }
+  } = useAragonApi()
+  if (token === null) return null
```

Comment:
This is all a bit weird to read. How about:

```js
const { api, appState: { github } } = useAragonApi()
if (!github || !github.token) return null
```

Then we'll have to use `github.token` elsewhere, but I think that's a good balance of readability here vs terseness there.